core/app/models/spree/return_authorization.rb

```diff
@@ -8,6 +8,7 @@ class ReturnAuthorization < Spree::Base
 
     belongs_to :stock_location
     belongs_to :reason, class_name: 'Spree::ReturnReason', foreign_key: :return_reason_id
+    belongs_to :recipient, class_name: Spree.user_class.to_s
```

Comment:
This should be `class_name: Spree::UserClassHandle.new`, which gets around some load order issues (which are likely causing the ci failure here)